eox_nelp/pearson_vue/rti_backend.py

```diff
@@ -50,5 +50,6 @@ def get_pipeline(self):
         Returns the RTI pipeline, which is a list of functions to be executed.
         """
         return [
+            check_completion_metadata,
```

Comment:
could you rebase and include the others pipes get_user_data get_exam_data, .....